core/templates/dev/head/components/forms/schema_editors/schema_based_list_editor_directive.html

```diff
@@ -25,7 +25,7 @@
 
   <!-- The margin-left is added here to left-align the button with the previous list items. -->
   <div style="height: 30px; margin-top: 4px;">
-    <button ng-show="!isOneLineInput || isAddItemButtonPresent"
+    <button ng-show="!isOneLineInput || isAddItemButtonPresent || showbutton()"
```

Comment:
Name should be showButton.

That said -- I think it's better to pick a name that explains what exactly the current showButton() method is checking, similar to the other names in the OR criterion. Alternatively, you could do `|| (localValue.length && localValue[localValue.length - 1].length)`.